examples/scenarios/supply_chain/rl/config.py

```diff
@@ -12,6 +12,6 @@
 distribution_features = ("remaining_order_quantity", "remaining_order_number")
 seller_features = ("total_demand", "sold", "demand")
 
-rl_algorithm = "dqn"  # 'dqn', 'ppo', or 'or'
+rl_algorithm = "or"  # 'dqn', 'ppo', or 'or'
```

Comment:
this config is under path `supply_chain/rl/...`, then could `or` be a kind of `rl algorithm`?

Could there be an independent config under `supply_chain/or/` ?